Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.cpp

```diff
@@ -651,131 +688,71 @@ void GenericConstraintProblem::gaussSeidel(SReal timeout, GenericConstraintSolve
     SReal *force = getF();
     SReal **w = getW();
     SReal tol = tolerance;
-
     SReal *d = _d.ptr();
 
-    int i, j, k, l, nb;
-
     SReal error=0.0;
-
     bool convergence = false;
     sofa::type::vector<SReal> tempForces;
-    if(sor != 1.0) tempForces.resize(dimension);
+
+    if(sor != 1.0)
+    {
+        tempForces.resize(dimension);
+    }
 
     if(scaleTolerance && !allVerified)
+    {
         tol *= dimension;
+    }
 
-    if(solver)
+    for(int i=0; i<dimension; )
     {
-        for(i=0; i<dimension; )
+        if(!constraintsResolutions[i])
         {
-            if(!constraintsResolutions[i])
-            {
-                msg_error(solver) << "Bad size of constraintsResolutions in GenericConstraintProblem" ;
-
-                dimension = i;
-                break;
-            }
-            constraintsResolutions[i]->init(i, w, force);
-            i += constraintsResolutions[i]->getNbLines();
+            msg_error("GenericConstraintSolver") << "Bad size of constraintsResolutions in GenericConstraintProblem" ;
```

Comment:
```suggestion
            msg_error(solver) << "Bad size of constraintsResolutions in GenericConstraintProblem" ;
```

Reply:
If you provide `solver` to `msg_error`, it will show the error message associated to solver object (i.e. the class `GenericConstraintSolver` and the name of the solver), whereas you provide only the class name.